pstoreds/addr_book.go

```diff
@@ -60,7 +82,7 @@ func NewAddrBook(ctx context.Context, ds ds.TxnDatastore, opts Options) (*dsAddr
 
 // Stop will signal the TTL manager to stop and block until it returns.
 func (mgr *dsAddrBook) Stop() {
```

Comment:
can we delete this? it's not part of the interface, right?

Reply:
Yeah, not part of the interface, which is an omission IMHO.